blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -801,26 +807,35 @@ def treat_bone(b, parent = None):
         for mesh in range (1, len(meshes)):
           for submesh in meshes[mesh].submeshes:
             submesh.bindtomesh(meshes[0])
+      checkPath(modelFilePath)#python does not create directory just the file. This function added to create t eh directory
       try:
         file = open(modelFilePath, 'w')
-      except IOError:
-        errmsg = "IOError " #%s: %s" % (errno, strerror)
+      except Exception as e:
+        print(str(e))
+        # errmsg = "IOError " #%s: %s" % (errno, strerror)
       buffer = skeleton.to_md5mesh(len(meshes[0].submeshes))
-      #for mesh in meshes:
       buffer = buffer + meshes[0].to_md5mesh()
       file.write(buffer)
       file.close()
       print( "saved mesh to " + modelFilePath )
 
 
-  if animationFilePath:
+  if animationFilePath:   
       #save animation file
+      if modelFilePath == None:
+        if len(meshes)>1:
+          for mesh in range (1, len(meshes)):
+            for submesh in meshes[mesh].submeshes:
+              submesh.bindtomesh(meshes[0])
       if len(ANIMATIONS)>0:
         anim = ANIMATIONS.popitem()[1] #ANIMATIONS.values()[0]
+        checkPath(animationFilePath)#python does not create directory just the file. This function added to create t eh directory
         try:
           file = open(animationFilePath, 'w')
-        except IOError:
-          errmsg = "IOError " #%s: %s" % (errno, strerror)
+        except Exception as e:
+          print(str(e))
+          # errmsg = "IOError " #%s: %s" % (errno, strerror)  

```

Comment:
Same as before: I think it would be best not to catch the exception and not to have any commented out code. The version history is for tracking old versions.